src/api/posts.tsx

```diff
@@ -0,0 +1,24 @@
+import { client } from '../utils/fetchClient';
+import { User } from '../types/User';
+import { Post } from '../types/Post';
+import { Comment, NewComment } from '../types/Comment';
+
+export const getUsers = () => {
+  return client.get<User[]>('/users');
+};
+
+export const getPosts = () => {
+  return client.get<Post[]>('/posts');
+};
+
+export const getComments = () => {
+  return client.get<Comment[]>('/comments');
+};
+
+export const addComments = (data: NewComment) => {
```

Comment:
```suggestion
export const addComment = (data: NewComment) => {
```
Actually, it adds only one comment